Doc/library/tempfile.rst

```diff
@@ -18,7 +18,7 @@ This module creates temporary files and directories.  It works on all
 supported platforms. :class:`TemporaryFile`, :class:`NamedTemporaryFile`,
 :class:`TemporaryDirectory`, and :class:`SpooledTemporaryFile` are high-level
 interfaces which provide automatic cleanup and can be used as
-context managers. :func:`mkstemp` and
+:term:`context manager`\s. :func:`mkstemp` and
```

Comment:
Something like this will link the whole "context managers" words:

```suggestion
:term:`context managers <context manager>`. :func:`mkstemp` and
```


For comparison, we have ``` :term:`named tuples <named tuple>` ``` elsewhere.